fallregister_backend/src/main/java/de/coronavirus/application/dtos/service/CityDTO.java

```diff
@@ -0,0 +1,13 @@
+package de.coronavirus.application.dtos.service;
+
+import de.coronavirus.domain.model.Street;
+
+import java.util.List;
+
+public interface CityDTO {
+
+    long getId();
+    String getName();
+    List<Long> getPostCodes();
```

Comment:
```suggestion
    List<PostCode> getPostCodes();
```

Bitte eine Liste von PostCodes zurückgeben